BuildForSDG/Team-037-Product-backend
# Ft sponsor view a farm

#### What does this PR do?
Tell us what this pull-request you are raising will add to the project.
A
![Capture63](https://user-images.githubusercontent.com/38535951/83912161-46d32680-a765-11ea-8a8b-c4d7832159ce.PNG)
![Capture64](https://user-images.githubusercontent.com/38535951/83912164-489cea00-a765-11ea-894f-06906ab5b241.PNG)

 a sponsor can view a farm that belongs to a farmer

#### How should this be manually tested?
Tell us how other people can test out this your feature to see if it's working as expected.
- git clone the repo
- npm i
- then run npm test or pm run dev

#### Any background context you want to provide?
Anything you'll like to say about this pull-request?

#### What are the relevant pivotal tracker stories?(if applicable)
Type in the pivotal-tracker ID of the task/ticket you completed and the link to that ticket.
[](url)

#### Screenshots
take pictures of the implementation of the project and paste them here


Please clear out all the descriptions, including this one as soon as you're done.
